roles/ipsec/templates/cluster.conf.j2

```diff
@@ -0,0 +1,14 @@
+{% for host in groups['all'] %}
```

Comment:
We have some sort of agreement that we would never refer to 'all'. If you want to include all hosts that participate in a Kubespray cluster, you want this filter:

```groups['k8s-cluster']|union(groups['etcd']|union(groups['calico-rr']|default([])|unique|sort
```

You may want to add in groups['vault'] but it's not included anywhere else